src/scss/06_components/navigation/_skip-link.scss

```diff
@@ -0,0 +1,42 @@
+.skip-link {
+  --border-width: 2px;
+  --border-color: var(--color-white);
+  --outline-width: 3px;
+  --background-color: var(--hdbt-color-black);
+  --color: var(--color-white);
+  --focus-outline-color: var(--hdbt-color-black);
+  word-wrap: normal;
+  background-color: var(--background-color);
+  border: var(--border-width) solid var(--border-color);
+  color: var(--color);
+  font-weight: $font-weight-medium;
+  left: 50%;
+  max-width: 90%;
+  position: absolute;
+  scroll-margin-top: 105px; // Amount of air that should be left when scrolled to top.
+  text-align: center;
+  text-decoration: none;
+  transform: translate(-50%, 0);
+  transition: transform ease-out 0.2s;
```

Comment:
I think we should add a "Prefer reduced motion" mediaquery here so that people who have trouble with animating things would not see the animation, but instead see the link just appear.

```SCSS
  @media (prefers-reduced-motion) {
    transition: none;
  }
```